.github/workflows/push.yml

```
@@ -183,7 +193,7 @@ jobs:
             -d zend_extension=opcache.so
             -d opcache.enable_cli=1
```

```suggestion
            -d opcache.enable_cli=1
            --asan
```
Or don't enable ASan in the configuration above.